makefiles/dependency_resolution.inc.mk

```diff
@@ -50,4 +50,13 @@ else
     $(shell $(COLOR_ECHO) "$(COLOR_RED)Deprecated modules are in use:$(COLOR_RESET)"\
                           "$(DEPRECATED_MODULES_USED)" 1>&2)
   endif
+  # Warn about used deprecated boards
+  include $(RIOTMAKE)/deprecated_boards.inc.mk
```

Comment:
Does this need to run here? Couldn't it be done in `Makefile.include`? Also can we make it only show up once with:

```
  # Do not warn when set from sub-make
  ifeq ($(MAKELEVEL),0)
    ...
  endif
```

Reply:
makes sense